setup

```diff
@@ -259,6 +275,9 @@ def test(args, remaining):
     if args.testname_regex is not None:
         testcmd += " -R {}".format(args.testname_regex)
 
+    if args.do_extra_test:
+        testcmd += " -C extra"
```

Comment:
Maybe also update the documentation that uses `-C extra` and change it to use this new flag instead:

https://shadow.github.io/docs/guide/developer_guide.html?highlight=extra#extra-tests  
https://shadow.github.io/docs/guide/getting_started_tor.html?highlight=extra#getting-started-tor

Reply:
Good catch; done